decompiler/util/options.py

```diff
@@ -144,6 +144,7 @@ def getlist(self, key: str, fallback: Optional[List[str]] = None) -> List[str]:
         else:
             if (value := self._settings_key_values.get(key, fallback)) is not None:
                 if isinstance(value, list):
+                    value = [x for x in value if x]
```

Comment:
Two questions:
1. Why can a value be None, 0, empty string. With `if x` you have to be careful.
2. Why do we need it now and not before?

Reply:
Now an empty string will be displayed, because it is a "valid" renaming option. 
But i think we should stay it like this, because it requires a user to type `""` in the config option....